__tests__/app.test.js

```diff
@@ -206,33 +206,33 @@ describe("APP", () => {
           });
       });
     });
-    describe('7/POST/api/articles/articleid/comment', () => {
+    describe("7/POST/api/articles/articleid/comment", () => {
       const commentToAdd = {
-        username: 'butter_bridge',
-        body: 'This article is amazing.  This has changed my life' 
-      }
-      test('responds with an object with the posted comment / correct properties', () => {
+        username: "butter_bridge",
+        body: "This article is amazing.  This has changed my life",
+      };
+      test("responds with an object with the posted comment / correct properties", () => {
         return request(app)
-        .post("/api/articles/1/comments")
-        .send(commentToAdd)
-        .expect(201)
-        .then(({body}) => {
-          expect(Object.keys(body.commentAdded)).toHaveLength(6)
-          expect(body.commentAdded).toHaveProperty("comment_id");
-          expect(body.commentAdded).toHaveProperty("article_id");
-          expect(body.commentAdded).toHaveProperty("body");
-          expect(body.commentAdded).toHaveProperty("author");
-          expect(body.commentAdded).toHaveProperty("created_at");
-          expect(body.commentAdded).toHaveProperty("votes")
-        })
-      })
+          .post("/api/articles/1/comments")
+          .send(commentToAdd)
+          .expect(201)
+          .then(({ body }) => {
+            expect(Object.keys(body.commentAdded)).toHaveLength(6);
+            expect(body.commentAdded).toHaveProperty("comment_id");
```

Comment:
given that you know the exact details of this posted test comment, it's more robust to expect the exact values of it. 

For checking the keys, maybe objectContaining will be helpful in more DRY code
[see the docs](https://jestjs.io/docs/expect#expectobjectcontainingobject)